recipes/mariadb-connector-c/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "mariadb-connector-c" %}
+{% set version = "3.2.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mariadb-corporation/mariadb-connector-c/archive/v{{ version }}.tar.gz
+  sha256: 6da2563a7940457846267989732b97028c350d7de07f89e355f15646641d0ed7
+
+build:
+  number: 0
+  missing_dso_whitelist:
+    - '*'
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - cmake
+    - ninja
+  host:
+    - openssl
+  run:
+    - mysql
+    
+test:
+  requires:
+    - {{ compiler('c') }}
+    - cmake
+    - ninja
+  commands: 
+    - mysql --help
+    - mysql_config --version
+
+about:
+  home: https://mariadb.com/kb/en/mariadb-connector-c/
+  license: LGPL-2.1-or-later
+  license_family: LGPL
+  license_file: COPYING.LIB
```

Comment:
Is this file not available in the source tarball?

Reply:
which file are you referring too?

Reply:
In that case you don't need to include it in the PR. Conda will find it from the source tarball and it's preferred to do that whenever possible.